src/docs/cookbook/networking/web-sockets.md

```diff
@@ -160,13 +163,11 @@ class _MyHomePageState extends State<MyHomePage> {
                 decoration: InputDecoration(labelText: 'Send a message'),
               ),
             ),
+            SizedBox(height: 24),
```

Comment:
I have removed the Padding and instead introduced a SizedBox here, so the example is a bit cleaner